src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python
+# pylint: disable=W0221
```

Comment:
Kenyi, just a standalone comment. I'd suggest not to disable any pylint checks otherwise we don't know what needs to be fixed in the future.

Reply:
@amaltaro Fair enough. 
The pylint message complains about something actually unchanged in this PR, but part of the SimpleCondorPlugin file:
```
Arguments number differs from %s method
```
Which is basically complaining about the number of arguments here:
https://github.com/dmwm/WMCore/blob/e94f8cd05b40b1293866914a79b444b57b2b48af/src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py#L411

vs here:
https://github.com/dmwm/WMCore/blob/e94f8cd05b40b1293866914a79b444b57b2b48af/src/python/WMCore/BossAir/Plugins/BasePlugin.py#L96

I considered the warning harmless enough, which is why I disabled it to make the "warning and error that must be fixed" go away. I could just create and `__init__` section defining `self. raiseEx` so the number of arguments match for example, if that sounds better.